internal/logging/logging.go

```diff
@@ -64,6 +66,9 @@ func SetupDefaultFileLogger(filename string, level slog.Level) {
 
 // SetupDefaultLogger sets up a default logger that writes to the specified writer
 func SetupDefaultLogger(w io.Writer, level slog.Level) {
-	h := slog.NewJSONHandler(w, &slog.HandlerOptions{Level: level})
+	h := tint.NewHandler(w, &tint.Options{
+		Level:      level,
+		TimeFormat: time.Kitchen,
+	})
```

Comment:
gives output like this: 
<img width="716" alt="Screenshot 2023-10-12 at 10 52 14" src="https://github.com/statechannels/go-nitro/assets/1833419/6ac944da-3673-4b6f-8f3e-b06ab286a194">